pkg/plugins/diff/delta_differ_test.go

```diff
@@ -0,0 +1,158 @@
+package tablediff
+
+import (
+	"context"
+	"errors"
+	"reflect"
+	"testing"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"google.golang.org/grpc"
+)
+
+func TestDeltaLakeDiffer_Diff(t *testing.T) {
+	tests := []struct {
+		name         string
+		client       TableDifferClientMock
+		diffResponse *DiffResponse
+		want         Response
+		err          error
+		expectedErr  error
+	}{
+		{
+			name:   "success",
+			client: TableDifferClientMock{},
+			diffResponse: &DiffResponse{
+				Entries: []*TableOperation{
+					{
+						Id:        "id1",
+						Timestamp: timestamppb.Now(),
+						Operation: "OPA",
+						Content: map[string]string{
```

Comment:
I guess that an updated version will include operation type?

Reply:
good catch